src/main/resources/assets/structurize/gui/windowstructurenameentry.xml

```diff
@@ -1,6 +1,6 @@
 <window xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../java/com/ldtteam/blockout/blockOut.xsd"
         pause="true" lightbox="true">
-    <label size="100% 11" pos="0 100" textalign="TOP_MIDDLE" color="white" label="$(com.ldtteam.structurize.gui.structure.edit.title)"/>
+    <label size="100% 11" pos="0 100" textalign="TOP_MIDDLE" color="white" label="Structure path:"/>
```

Comment:
Why shouldn't this be a translation string?

Reply:
I was dead translation, I was just testing things, need to add it into lang first